src/_P103_Atlas_EZO_pH_ORP_EC.ino

```diff
@@ -65,22 +67,22 @@ boolean Plugin_103(uint8_t function, struct EventStruct *event, String &string)
     break;
   }
 
-    #if USE_I2C_DEVICE_SCAN
-    case PLUGIN_I2C_GET_ADDRESSES_HEX:
+    case PLUGIN_I2C_HAS_ADDRESS:
+    case PLUGIN_WEBFORM_SHOW_I2C_PARAMS:
     {
-      string = F("62,63,64"); // List of device addresses, hex, comma separated, _no_ 0x prefix
-      success = true;
+      const int i2cAddressValues[] = {0x62, 0x63, 0x64, 0x65, 0x66, 0x67};
+      if (function == PLUGIN_WEBFORM_SHOW_I2C_PARAMS) {
+        addFormSelectorI2C(F("plugin_103_i2c"), _P103_ATLASEZO_I2C_NB_OPTIONS, i2cAddressValues, PCONFIG(1));
+        addFormNote(F("pH: 0x63, ORP: 0x62, EC: 0x64. The plugin is able to detect the type of device automatically."));
```

Comment:
What about the other 3 addresses?

Reply:
We should ask the original author about that.

Reply:
* Atlas EC: From chemical production to hydroponics, conductivity is a fundamental measurement of water chemistry. The Atlas Scientific EZO-EC Circuit reads conductivity, salinity, and TDS (PPM).
* Atlas RTD: the Atlas Scientific EZO-RTD Circuit makes taking high accuracy readings from a platinum RTD probes easy. The EZO-RTD Circuit can work with any class of 2, 3, or 4 wire platinum RTD probes and has a sensing range of -126.000 °C to +1,254 °C.
* Atlas PMP: Dosing pump, controlled via a.o. I2C.

These options are not supported by the plugin, not sure if that is intentional, but more likely it wasn't needed at the time of implementation 😃 
Maybe we should limit the address range to the first 3?

Reply:
If not supported, then I guess you better remove them from the list.
Maybe include this (or a link to this comment) as a remark next to it for future implementations.

Reply:
Updated the list to only show the supported devices, to support other devices, the code needs more changes.

Reply:
0x62: Default ORP I2C address
0x63: Default pH I2C address
0x64: Default EC I2C address

But: Each device could be programmed to an I2C address between 0x01 an 0x7F
So for me the defaults are fine. I use them. If any one wants to use another address, we have to figure out, how this could work.

Reply:
Atlas DO: If anybody wants this device, it will be no probelm to add it to the plugin. As i do not have and need this device, I am not able to to this at the moment. But if any body wants it, I will be able to code it, if he sends me the device for testing purpose. It is not a big thing.